src/main/java/buildcraftAdditions/items/ItemBase.java

```diff
@@ -19,25 +14,11 @@
  */
 
 public class ItemBase extends Item {
-	public IIcon icon;
-	public String name;
```

Comment:
You can access itemIcon and if you really want to access the name you can get a substring of the texture name or the unlocalized name.